### PR TITLE
Bump TESTed stuff

### PR DESCRIPTION
**Pull request by niknetniko**
See https://github.com/dodona-edu/universal-judge/pull/511.

### CODE REVIEW

File: dodona-tested.dockerfile [hunk 1]
```diff
@@ -17,41 +20,44 @@ RUN mkdir -p /usr/share/man/man1mkdir -p /usr/share/man/man1 \
...
+       # Dependencies for GHCup
+       autoconf build-essential zlib1g-dev libgmp-dev \
  && curl https://packages.microsoft.com/config/debian/11/packages-microsoft-prod.deb --output packages-microsoft-prod.deb \
  && dpkg -i packages-microsoft-prod.deb \
  && rm packages-microsoft-prod.deb \
- # Add nodejs v18
- && bash -c 'set -o pipefail && curl -fsSL https://deb.nodesource.com/setup_18.x | bash -' \
- # Install programming languages \
+ # JavaScript dependencies
+ && bash -c 'set -o pipefail && curl -fsSL https://deb.nodesource.com/setup_21.x | bash -' \
```
Comment (bmesuere) [inline dodona-tested.dockerfile:29]: node 21 is no lts release, but 22 is out already

**Review by bmesuere — APPROVED** [review]

File: dodona-tested.dockerfile (hunk 1 above)
Comment (pdawyndt) [reply dodona-tested.dockerfile:29]: There was already a question by one student in this respect, so updating to 22 would indeed be a good idea towards nexr academic year.

Comment (niknetniko) [reply dodona-tested.dockerfile:29]: Ah right, I read the wrong line on https://nodejs.org/en/about/previous-releases, I meant to use the latest LTS, which is indeed 22.

**Comment by niknetniko** [comment]
(I have no more patience for the R images, so merging now)

**Pull request merged**